src/components/footer/Pager.ts

```diff
@@ -112,7 +112,7 @@ export class DataTablePager {
 
       this.onPaged.emit({
         type: 'pager-event',
-        value: page
+        page: page
```

Comment:
We should use `value` instead as there are other places that are looking for that.